backend/inspirehep/search/facets.py

```diff
@@ -174,10 +174,10 @@ def records_hep():
         "aggs": {
             **hep_earliest_date_aggregation(order=1),
```

Comment:
why not do
```
from itertools import count
order = count(start=1)
```
then
```suggestion
            **hep_earliest_date_aggregation(order=next(order)),
```
etc., instead of hardcoding the order?